src/autopas/pairwiseFunctors/Functor.h

```diff
@@ -209,6 +209,14 @@ class Functor {
    */
   virtual bool allowsNonNewton3() = 0;
 
+  /**
+   * Specifies whether the functor can handle mixed functor-calls with newton3 enabled and disabled in one iteration,
+   * while correctly calculating the globals after an iteration
+   * @return true if and only if this functor can handle mixed functor-calls with newton3 enabled and disabled in one
+   * iteration
+   */
```

Comment:
done in [4a5aece](https://github.com/AutoPas/AutoPas/pull/745/commits/4a5aece0f8222be36761514e6c229ecbf982c33f), since allowsMixedNewton3() is removed again.